tests/lib/ignored-paths.js

```diff
@@ -127,14 +127,14 @@ describe("IgnoredPaths", () => {
             const ignorePatterns = getIgnorePatterns(ignoredPaths);
 
             assert.isNotNull(ignoredPaths.baseDir);
-            assert.equal(getIgnoreFiles(ignoredPaths), expectedIgnoreFile);
+            assert.deepEqual(getIgnoreFiles(ignoredPaths), [expectedIgnoreFile]);
```

Comment:
Similarly, by asserting on length here, we could also do a strict comparison of the first element.